projects/ngx-sentry/src/lib/sentry.interceptor.ts

```diff
@@ -1,6 +1,6 @@
 import { Injectable, Inject } from '@angular/core'
 import { HttpInterceptor, HttpRequest, HttpHandler, HttpErrorResponse } from '@angular/common/http'
-import { captureMessage } from '@sentry/browser/esm'
+import * as Sentry from '@sentry/browser'
```

Comment:
Please stick with the destructuring imports `{ ... }` as they're more expressive which functions are used in this file.

Reply:
ok done